src/main/java/core/basesyntax/Main.java

```diff
@@ -0,0 +1,15 @@
+package core.basesyntax;
+
+public class Main {
+    private static final int NUMBER_OF_FIGURES_IN_EACH_OF_TWO_GROUPS = 3;
+
+    public static void main(String[] args) {
+        FigureSupplier figureSupplier = new FigureSupplier();
+        for (int i = 0; i < NUMBER_OF_FIGURES_IN_EACH_OF_TWO_GROUPS; i++) {
+            figureSupplier.getRandomFigure().draw();
+        }
+        for (int i = 0; i < NUMBER_OF_FIGURES_IN_EACH_OF_TWO_GROUPS; i++) {
+            figureSupplier.getDefaultFigure().draw();
+        }
+    }
```

Comment:
Please pay attention to condition of the task:
In the main() method we need to create an array of figures (the size of array can be 3 or 6, it doesn't matter). The first half of figures in this array should be generated with random parameters.